aspnetcore/security/enforcing-ssl.md

```diff
@@ -343,6 +343,43 @@ The following command provides help on the `dev-certs` tool:
 dotnet dev-certs https --help
 ```
 
+<a name="trust-ff"></a>
+
+### Trust the HTTPS certificate on Firefox
+
+Create a policy file at:
+
+* On windows: `%PROGRAMFILES%\Mozilla Firefox\distribution\policies.json`
+* On MacOS: `Firefox.app/Contents/Resources/distribution`
+
+Add the following JSON to the Firefox policy file:
+
+```json
+{
+  "policies": {
+    "Certificates": {
+      "ImportEnterpriseRoots": true
+    }
+  }
+}
+```
+
+The preceding policy file makes Firefox trust certificates from the trusted certificates in the Windows certificate store. The next section provides an alternative approach for the Firefox browser.
+
+### Firefox SEC_ERROR_INADEQUATE_KEY_USAGE certificate error
```

Comment:
I don't remember this bit being required.

The json policy above is doing precisely this.

Reply:
The browser approach is much simpler and works on all platforms. Can I put the browser approach first? See how I've done it. I can remove the browser approach if you think that's best.